pkg/modules/libreoffice/api/api.go

```diff
@@ -69,6 +69,10 @@ type Options struct {
 	// Optional
 	ReduceImageResolution bool
 
+	// ExportNotesInMargin allows to export comments in margin
+	// Optional
+	ExportNotesInMargin bool
```

Comment:
```suggestion
	// ExportNotesInMargin allows to export comments in margin.
	// Optional
	ExportNotesInMargin bool
```

Reply:
Could you also move this before `LosslessImageCompression`?

Reply:
Done